compiler/src/utils/warnings.re

```diff
@@ -119,10 +119,12 @@ let message =
     )
   | NonClosedRecordPattern(s) =>
     "the following fields are missing from the record pattern: " ++ s
-  | FuncWasmUnsafe(func) =>
+  | FuncWasmUnsafe(func, m) =>
     "it looks like you are using "
     ++ func
-    ++ " on two unsafe Wasm values here.\nThis is generally unsafe and will cause errors. Use one of the equivalent functions in `WasmI32`, `WasmI64`, `WasmF32`, or `WasmF64` instead."
+    ++ " on two unsafe Wasm values here.\nThis is generally unsafe and will cause errors. Use "
+    ++ m
+    ++ " instead."
```

Comment:
You still need to tell them that they need to use one of the functions in that library. I think this phrasing would be pretty confusing if you don't have the context that libraries exist for these types.

Reply:
Agreed, now it says the module as well